src/api/java/mekanism/api/radiation/IRadiationManager.java

```diff
@@ -114,30 +130,54 @@ public interface IRadiationManager {
      * Gets an unmodifiable table of the radiation sources tracked by this manager. This table keeps track of radiation sources on both a chunk and position based level.
      *
      * @return Unmodifiable table of radiation sources.
+     * @deprecated please tell us how you use this so we can provide an alternative
```

Comment:
[This](https://github.com/Frinn38/Custom-Machinery-Mekanism/blob/1.21/src/main/java/fr/frinn/custommachinerymekanism/common/component/RadiationMachineComponent.java#L35) seems to be the [only](https://waifu.neoforged.net/d/aec29a5ymel1cc/waifu-classes?orgId=1&from=2025-01-01T06:00:00.000Z&to=2026-01-01T05:59:59.999Z&timezone=browser&var-version=1.21.1-neoforge&var-class=mekanism%2Fapi%2Fradiation%2FIRadiationManager&var-method=getRadiationSources&var-field=.%2A&viewPanel=panel-7-clone-0) caller of this method in 1.21.1

Reply:
provided an alternative